source/math/FloatConvexHull2D.ooc

```diff
@@ -21,55 +21,122 @@ import FloatTransform2D
 import text/StringTokenizer
 
 FloatConvexHull2D: class {
-	// Points must be in clockwise order
-	// TODO: Implement computation of convex hull from a set of points
 	_points: VectorList<FloatPoint2D>
+	_hull: VectorList<FloatPoint2D>
 	points ::= this _points
+	hull ::= this _hull
 	count ::= this _points count
```

Comment:
Wouldn't it make more sense to have the property `count` return `this _hull count` now instead? Or is it even necessary to keep the point set saved in the class at all? Can't the hull points just be saved to the `points` list?